contracts/Example1.sol

```diff
@@ -2,6 +2,6 @@ pragma solidity ^0.8.4;
 
 contract Example1 {
     uint8 public day = 10;
-    uint256 public amount = 1000;
+    uint32 public amount = 1000;
```

Comment:
It would be wrong to assume the amount will always fit into `uint32`. Here we should move `address` with `uint8` so that they are packed together

Reply:
Hey @shubham-kanodia, I thought that addresses took 32 byte slots i.e. regardless of the ordering it would each have taken one slot it.

Reply:
They only occupy 20 bytes of a slot so a uint8 can easily be fit in the remaining 96 bits